waspc/data/Generator/templates/sdk/wasp/server/auth/oauth/provider.ts

```diff
@@ -1,23 +1,19 @@
 import { OAuth2Provider, OAuth2ProviderWithPKCE } from "arctic";
 
 export function defineProvider<
-  OAuthClient extends OAuth2Provider | OAuth2ProviderWithPKCE,
-  Env extends Record<string, string>
+  OAuthClient extends OAuth2Provider | OAuth2ProviderWithPKCE
 >({
   id,
   displayName,
-  env,
   oAuthClient,
 }: {
   id: string;
   displayName: string;
-  env: Env;
```

Comment:
Instead of OAuth env vars having their special validation and living in a special place, they are now used directly from the `env` object.